specs/0107-pretranslated-status.md

```diff
@@ -0,0 +1,48 @@
+- Feature Name: Pretranslated status
+- Created: 2020-09-21
+- Associated Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1666320
+
+# Summary
+
+Introduce a new translation status for Pretranslated strings to distinct them from Fuzzy strings.
```

Comment:
nit: differentiate/distinguish? I don't think *distinct* is a verb. Same later (e.g. line 25).